packages/server-wallet/src/wallet/store.ts

```diff
@@ -236,7 +236,7 @@ export class Store {
       } = objective;
       const channelId = calculateChannelId(signedState);
       const singedStateWithHash = {...signedState, stateHash: hashState(signedState)};
-      validateSignatures(singedStateWithHash);
+      // validateSignatures(singedStateWithHash); // TODO do we need this?
```

Comment:
I have put the validation back in inside `addObjective`. I don't see a reason why we _shouldn't_ validate the signatures here when we are elsewhere. There is likely an attack vector on the wallet otherwise, where I can put forged states (with incorrect signatures) in your wallet by bundling them inside an objective.